### PR TITLE
tile index

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Video_Display.md [hunk 1]
```diff
@@ -633,13 +633,17 @@ affects the priority ordering, thus other sprites with lower priority may be
...
+### Byte 2 - Tile Index
+
+In 8x8 mode (LCDC bit 2 = 0), this byte specifies the sprite's only tile index ($00-$FF).
+This unsigned value selects a tile from the memory area at $8000-$8FFF.
+In CGB Mode this could be either in
+VRAM bank 0 or 1, depending on bit 3 of the following byte.
+In 8x16 mode (LCDC bit 2 = 1), the memory area at $8000-$8FFF is still interpreted
+as a series of 8x8 tiles, where every 2 tiles form a sprite. In this mode, this byte
+specifies the index of the first (top) tile of the sprite. This is enforced by the
+hardware: the least significant bit of the tile index is ignored. IE: the top 8x8
```
Comment (ISSOtm) [inline content/Video_Display.md:645]: Please replace "IE" with "I.e." or "That is", so as not to confuse it for the register `IE`.

File: content/Video_Display.md [hunk 2]
```diff
@@ -633,13 +633,17 @@ affects the priority ordering, thus other sprites with lower priority may be
...
+tile is "NN AND $FE", and the bottom 8x8 tile is "NN OR $01".
```
Comment (ISSOtm) [inline content/Video_Display.md:646]: Please use binary operators `&` and `|` instead of words, to make it clearer.

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Video_Display.md (hunk 2 above)
Comment (avivace) [reply content/Video_Display.md:646]: If using the binary operators, the string should not be "A AND B" but `A & B`

Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**